GOVERNANCE.md

```diff
@@ -0,0 +1,147 @@
+* [Technical Steering Committee (TSC)](#Technical-Steering-Committee) 
+  members provide overall high-level project guidance.
+
+
+## Contributors
+
+The Rez project grows and thrives from assistance from
+Contributors. Contributors include anyone in the community that
+submits code, documentation, or other technical artifacts to the
+project. However, such contributions must be approved by a project
+Committer before they become a part of the project.  
+
+Anyone can be a Contributor. You need no formal approval from the
+project, beyond the legal forms.
+
+### How to Become a Contributor
+
+* Review the coding standards in `CONTRIBUTING.md` to ensure your contribution
+  is in line with the project's coding and styling guidelines.
+
+* Sign the Individual CLA, or have your organization sign the Corporate CLA.
+
+* Submit your code as a PR with the appropriate DCO sign-off.
+
+## Committers
+
+Project Committers have merge access on the Rez GitHub repository
+and are responsible for approving submissions by Contributors.
+
+### Committer Responsibilities
+
+Typical activities of a Committer include:
+
+* Helping users and novice contributors.
+
+* Ensuring a response to questions posted to the ASWF rez slack channel.
+
+* Contributing code and documentation changes that improve the project.
+
+* Reviewing and commenting on issues and pull requests.
+
+* Ensuring that changes and new code meet acceptable standards and are
+  in the long-term interest of the project.
+
+* Participation in working groups.
+
+* Merging pull requests.
+
+### How to Become a Committer
+
+Any existing Committer can nominate an individual making significant
+and valuable contributions to the Rez project to become a new
+Committer. New committers are approved by vote of the TSC.
+
+If you are interested in becoming a Committer, contact the TSC at:
+  rez-discussion@lists.aswf.io
+
+## Technical Steering Committee
+
+The Technical Steering Committee (TSC) oversees the overall technical
+direction of Rez, as defined in the project
+[charter](https://github.com/AcademySoftwareFoundation/foundation/blob/main/project_charters/rez-charter.pdf).
+This charter defines the TSC member terms and succession policies.
+
+The responsibilities of the TSC include:
+
+* Coordinating technical direction of the project.
+
+* Project governance and contribution policy.
+
+* GitHub repository administration.
+
+* Maintaining the list of additional Committers.
+
+* Appointing representatives to work with other open source or open
+  standards communities.
+
+* Discussions, seeking consensus, and where necessary, voting on
+  technical matters relating to the code base that affect multiple
+  projects.
+
+* Coordinating any marketing, events, or communications regarding the
+  project.
+
+The TSC elects a Chair person, who acts as the project manager,
+organizing meetings and providing oversight to project
+administration. The Chair is elected by the TSC. The Chair may
```

Comment:
Should we allow the chair to delegate all of these?

Reply:
adjusted with respect to comment.